src/conways-game-of-life/index.ts

```diff
@@ -0,0 +1,81 @@
+const updateCellStatus = (x: number, y: number, currentCellStatus: number, neighbourSum: number) => {
+    if (currentCellStatus === 1 && neighbourSum > 3) {
+        return 0;
+    }
+    else if (currentCellStatus === 1 && neighbourSum < 2) {
+        return 0;
+    }
+    else if (currentCellStatus === 0 && neighbourSum === 3) {
+        return 1;
+    }
+    else {
+        return currentCellStatus;
+    }
```

Comment:
You don't need the `else` on these conditions. Think about it, if the first condition match, you return a value and don't execute the second block. If the second condition match, you don't execute the third block. If you going with this logic at the end you don't return the current status.

Reply:
It makes sense. Else removed.